zone/command.cpp

```diff
@@ -387,6 +387,7 @@ int command_init(void)
 		command_add("unscribespells", "- Clear out your or your player target's spell book.", AccountStatus::GMCoder, command_unscribespells) ||
 		command_add("untraindisc", "[spellid] - Untrain specified discipline from your target.", AccountStatus::GMCoder, command_untraindisc) ||
 		command_add("untraindiscs", "- Untrains all disciplines from your target.", AccountStatus::GMCoder, command_untraindiscs) ||
+		command_add("updatechecksum", "gm only", 250, command_updatechecksum) ||
```

Comment:
May want to use the AccountStatus:: Function instead of Magic Number here

Reply:
Done.